java/apps/authentication-portal/src/main/webapp/includes/layout-resolver.jsp

```diff
@@ -16,37 +16,98 @@
   ~ under the License.
 --%>
 
-<%@ page import="java.util.Map" %>
-<%@ page import="java.util.HashMap" %>
-
-<%-- Change the layout name to activate another layout --%>
+<%-- Layout Resolver --%>
 <%
-    String layout = "default";
-%>
 
-<%-- Activate the "custom" layout if exists --%>
-<%
-    if (config.getServletContext().getResource("extensions/layouts/custom/body.ser") != null) {
-        layout = "custom";
+    // Get the layout store url from the configuration file
+    String tempLayoutStoreURL = application.getInitParameter("LayoutStoreURL");
+    if (!StringUtils.isBlank(tempLayoutStoreURL)) {
+        layoutStoreURL = tempLayoutStoreURL;
     }
-%>
 
-<%-- Layout Resolving Part --%>
-<%
-    String layoutFileRelativePath;
-    Map<String, Object> layoutData = new HashMap<String, Object>();
-
-    if (!layout.equals("custom")) {
-        if (layout.equals("default")) {
-            layoutFileRelativePath = "includes/layouts/" + layout + "/body.ser";
-        } else {
-            layoutFileRelativePath = "extensions/layouts/" + layout + "/body.ser";
-            if (config.getServletContext().getResource(layoutFileRelativePath) == null) {
-                layout = "default";
-                layoutFileRelativePath = "includes/layouts/default/body.ser";
+    // Common data for the layout file.
+    // temporarily commented during unification
+    // layoutData.put("BASE_URL", layoutStoreURL.replace("${tenantDomain}", tenantRequestingPreferences));
```

Comment:
Check whether this can be uncommented now

```suggestion
    // layoutData.put("BASE_URL", layoutStoreURL.replace("${tenantDomain}", tenantRequestingPreferences));
```

Reply:
Fixed with 6af8d14